[2024-02-20T20:36:09.355+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_dag.save_posts manual__2024-02-20T20:36:03.736529+00:00 [queued]>
[2024-02-20T20:36:09.359+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_dag.save_posts manual__2024-02-20T20:36:03.736529+00:00 [queued]>
[2024-02-20T20:36:09.359+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-20T20:36:09.365+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): save_posts> on 2024-02-20 20:36:03.736529+00:00
[2024-02-20T20:36:09.368+0000] {standard_task_runner.py:60} INFO - Started process 9308 to run task
[2024-02-20T20:36:09.370+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'save_posts', 'manual__2024-02-20T20:36:03.736529+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/rest_api.py', '--cfg-path', '/tmp/tmp34aqwwv1']
[2024-02-20T20:36:09.372+0000] {standard_task_runner.py:88} INFO - Job 27: Subtask save_posts
[2024-02-20T20:36:09.395+0000] {task_command.py:423} INFO - Running <TaskInstance: api_dag.save_posts manual__2024-02-20T20:36:03.736529+00:00 [running]> on host 4b7631aaf19c
[2024-02-20T20:36:09.430+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_dag' AIRFLOW_CTX_TASK_ID='save_posts' AIRFLOW_CTX_EXECUTION_DATE='2024-02-20T20:36:03.736529+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-20T20:36:03.736529+00:00'
[2024-02-20T20:36:09.432+0000] {rest_api.py:17} ERROR - Failed to save file: [Errno 2] No such file or directory: '/Users/princepriyadarshi/Downloads/Smaple pro/***/dags/posts.json'
[2024-02-20T20:36:09.433+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-20T20:36:09.437+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=save_posts, execution_date=20240220T203603, start_date=20240220T203609, end_date=20240220T203609
[2024-02-20T20:36:09.463+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-20T20:36:09.472+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
